community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/idmapping/string/EncodingIdMapper.java

```diff
@@ -82,7 +82,9 @@
     private boolean readyForUse;
     private long[][] sortBuckets;
     private long size;
-    private final List<IdGroup> idGroups = new ArrayList<>();
+
+    private final IdGroup[] idGroups = new IdGroup[10];
```

Comment:
Just an arbitrary number that most imports will be within range of. I meant to have it grow on demand in `put`, but forgot. I'll fix that